yadro/yadro.cpp

```diff
@@ -30,10 +30,15 @@
 #include "yadro.hpp"
 #include "yadro_sensorstables.hpp"
 #include "sensors.hpp"
+#include "versions.hpp"
 #include "watcher.hpp"
 
 static yadrowatcher watcher;
+
 constexpr oid hostPowerState_oid[] = {1, 3, 6, 1, 4, 1, 49769, 1, 1};
+constexpr oid versionBMC_oid[] = {1, 3, 6, 1, 4, 1, 49769, 3, 1, 1};
```

Comment:
I would suggest replacing this repeating `1, 3, 6, 1, 4, 1, 49769` with a macro like `YADRO_OID`.
Maybe even a function-like variadic macro like this:
```
#define YADRO_OID(args...) {1, 3, 6, 1, 4, 1, 49769, ##args}
...
constexpr oid versionBMC_oid[] = YADRO_OID(3, 1, 1);
```